src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033015 "retinal astrocyte")
 EquivalentClasses(obo:CL_4033015 ObjectIntersectionOf(obo:CL_0000127 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000966)))
 
+# Class: obo:CL_4033016 (secondary crest myofibroblast)
+
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033016 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033016 "2023-03-16T09:17:47Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31331942") oboInOwl:hasExactSynonym obo:CL_4033016 "secondary-crest myofibroblast")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33624948") oboInOwl:hasRelatedSynonym obo:CL_4033016 "SCMF")
```

Comment:
Recommend adding synonym type abbreviation where appropriate.

Reply:
Thank you, I wil apply this comment for the rest of the terms

Reply:
Please annotate the synonym using the has_synonym_type annotation property.
See annotation on 'medullary reticular cell' as an example.

Reply:
Thank you!